pyccel/codegen/printing/fcode.py

```diff
@@ -206,6 +212,37 @@ def __init__(self, parser, prefix_module = None):
 
         self.prefix_module = prefix_module
 
+    def check_restrictions(self):
+        """Checks if the namespace contains any of the shortcuts"""
+        valid = [True] * len(iso_c_binding_shortcuts)
+        for variableName in self._namespace.variables:
+            for idx, alias in enumerate(iso_c_binding_shortcuts):
+                if variableName == alias:
+                    valid[idx] = False
+        for functionName in self._namespace.functions:
+            for idx, alias in enumerate(iso_c_binding_shortcuts):
+                if functionName == alias:
+                    valid[idx] = False
+        for variableName in self._namespace.imports['variables']:
+            for idx, alias in enumerate(iso_c_binding_shortcuts):
+                if variableName == alias:
+                    valid[idx] = False
+        for functionName in self._namespace.imports['functions']:
+            for idx, alias in enumerate(iso_c_binding_shortcuts):
+                if functionName == alias:
+                    valid[idx] = False
+        return valid
+
+    def print_macros(self):
+        """Prints the use line that indicates the macros used"""
+        macro = "use ISO_C_BINDING, only: "
+        rename = []
+        for idx, used in enumerate(self._macros):
+            if used:
+                rename.append(iso_c_binding_shortcuts[idx] + ' => ' + iso_c_bindings[idx])
+        macro += " , ".join(rename)
+        return macro
```

Comment:
`C_INT`, etc are not macros, they are named constants (see [here](https://stackoverflow.com/tags/fortran-iso-c-binding/info) for more information). However here, you are not printing named constants either, you are printing the import of named constants. It is important to try and find a clear name so it is clear what is happening when the function is used elsewhere in the code. This means that other developers don't need to find the function and read the docstring to understand what is going on. For example this function could be called `_print_kind_imports`

Reply:
noted! i will change it!